packages/next-urql/src/types.ts

```diff
@@ -40,6 +45,7 @@ export interface WithUrqlClient {
 
 export interface WithUrqlProps extends WithUrqlClient, WithUrqlState {
   resetUrqlClient?: () => void;
+  pageProps: any;
```

Comment:
I'm not sure why the index signature `[key: string]: any` didn't seem to be sufficient here.